ts_scripts/install_dependencies.py

```diff
@@ -50,18 +50,21 @@ def install_python_packages(self, cuda_version, requirements_file_path, nightly)
             # as it may reinstall the packages with different versions
             os.system("conda install -y conda-build")
 
-        self.install_torch_packages(cuda_version)
         os.system(f"{sys.executable} -m pip install -U pip setuptools")
         # developer.txt also installs packages from common.txt
         os.system(f"{sys.executable} -m pip install -U -r {requirements_file_path}")
-        # If conda is available install conda-build package
 
-        # TODO: This will run 2 installations for torch but to make this cleaner we should first refactor all of our requirements.txt into just 2 files
-        # And then make torch an optional dependency for the common.txt
+        # Install dependencies for GPU
+        if cuda_version:
+            os.system(f"{sys.executable} -m pip install -U -r gpu.txt")
```

Comment:
might need to link to gpu file path in the requirements folder since file isnt here

But taking a step back what does is the requirements gpu that you're thinking of having? As in you need to pick a default CUDA version.

I think it might make sense to remove torch as a dependency in our requirements.txt so users need to first explicitly install it and then in our CI and docs we default to 11.8 for example. Doing this will let us all delete all our all requirements.txt